internal/build/gotool.go

```diff
@@ -55,7 +55,7 @@ func (g *GoToolchain) Go(command string, args ...string) *exec.Cmd {
 	}
 	// CKZG by default is not portable, append the necessary build flags to make
 	// it not rely on modern CPU instructions and enable linking against
-	tool.Env = append(tool.Env, "CGO_CFLAGS=-D__BLST_PORTABLE__")
+	tool.Env = append(tool.Env, "CGO_CFLAGS=-std=c99 -O2 -D__BLST_PORTABLE__")
```

Comment:
I'm adding this change as an attempt to fix this build failure: https://launchpadlibrarian.net/681424348/buildlog_ubuntu-trusty-amd64.ethereum-unstable_1.13.0+build28791+trusty_BUILDING.txt.gz